template.yaml

```diff
@@ -934,6 +962,299 @@ Resources:
             Unit: Count
   #endregion
 
+  #region Authentication Callback Lambda
+
+  AuthenticationCallbackFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_116: DLQ is not appropriate for a Lambda invoked by an API
+    # checkov:skip=CKV_AWS_117: VPC settings are global
+    Properties:
+      FunctionName: !Sub ${Environment}-AuthenticationCallbackFunction
+      AutoPublishAlias: latest
+      CodeUri: ./oidc-api
+      Handler: uk.gov.di.authentication.oidc.lambda.AuthenticationCallbackHandler::handleRequest
+      ReservedConcurrentExecutions: 1
+      LoggingConfig:
+        LogGroup: !Ref AuthenticationCallbackFunctionLogGroup
+      Environment:
+        Variables:
+          # checkov:skip=CKV_AWS_173: These environment variables do not require encryption.
+          ENVIRONMENT: !Sub ${Environment}
+          ACCOUNT_INTERVENTION_SERVICE_ACTION_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_CALL_ENABLED: true
+          ACCOUNT_INTERVENTION_SERVICE_URI: !Ref AccountInterventionsServiceUriSecret
+          AUTHENTICATION_BACKEND_URI: !Sub
+            - https://${AuthExternalApiId}-${VpcId}.execute-api.eu-west-2.amazonaws.com/${AuthEnvironment}/
+            - AuthExternalApiId: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authExternalApiId ]
+              VpcId: !ImportValue vpc-ExecuteApiGatewayEndpointId
+              AuthEnvironment: !FindInMap [ EnvironmentConfiguration, !Ref Environment, authEnvironment ]
+          ACCOUNT_INTERVENTIONS_ERROR_METRIC_NAME: "AISException"
```

Comment:
Where did you get this from? I can't seem to find it being set in the old lambda

Reply:
Its set as the default in the variables.tf file

Reply:
Or are you just talking about the ENV variable itself ? https://github.com/govuk-one-login/authentication-api/blob/main/ci/terraform/oidc/authentication-callback.tf#L37

Reply:
Yes completely missed that 🤦 sorry!